components/tables/characterTable/characterTable.module.css

```diff
@@ -4,4 +4,26 @@
     display: flex;
     align-items: baseline;
     justify-content: space-between;
+}
+
+.characterRow {
+    margin-bottom: 10px;
+}
+
+.characterField, .profilePic {
+    padding-top: 10px;
+    padding-bottom: 10px;
+    vertical-align: middle;
+}
+
+.profilePic {
```

Comment:
What's the difference between `profilePic` and `profileImage` ? Are they totally seperate components?

If it's more that profilePics are in characterTables, could something like BEM help? http://getbem.com/introduction/
`characterTable__profilePic` instead of `profilePic` standalone

Reply:
Hadn't heard of BEM before, I see the benefits to using it but I'm not sure what the changes I should make to this MR are?

They are the same thing, I suppose I should be making this a `ProfileImageComponent`? Is this BEM? It's not clear to me which part is React design and which is CSS design

Reply:
I guess this is resolved by my previous commit which deletes the `profileImage` class (in favour of Chakra props) but it would be good to sort out the CSS at some point anyway I think :sweat_smile: I'll research it, at some point... :P